centos.org/pipelines/lib/foremanDuffyJob.groovy

```diff
@@ -66,22 +81,23 @@ pipeline {
                     """
 
                     writeFile(file: 'otel_env', text: otel_env)
-                    duffy_scp_in('otel_env', 'otel_env', 'duffy_box', './')
-                    sh(script: 'rm -rf otel_env', label: 'remove otel_env file')
 
                     def branches = [:]
                     pipelines.each { action, oses ->
                         oses.each { os ->
                             def name = "${os}-${action}"
+                            def username = "pipe-${name}"
```

Comment:
On line 120 you have this:
```suggestion
                            def username = "pipe-${name}-${action}"
```

Reply:
No, there it is `pipe-${os}-${action}` (as `$name` is different there)
but I agree it's inconsistent.